google/cloud/forseti/enforcer/enforcer.py

```diff
@@ -37,58 +33,46 @@
 from google.cloud.forseti.enforcer import enforcer_log_pb2
 
 
-# Hack to make the test pass due to duplicate flag error here
-# and inventory_loader.
-# TODO: Find a way to remove this try/except, possibly dividing the tests
-# into different test suites.
-try:
-    flags.DEFINE_string(
-        'forseti_config',
-        '/home/ubuntu/forseti-security/configs/forseti_conf_server.yaml',
-        'Fully qualified path and filename of the Forseti config file.')
-except flags.DuplicateFlagError:
-    pass
-
-flags.DEFINE_string('enforce_project', None,
-                    'A single projectId to enforce the firewall on. Must be '
-                    'used with the policy_file flag.')
-
-flags.DEFINE_string('policy_file', None,
-                    'A json encoded policy file to enforce, must contain a '
-                    'list of Firewall resources to apply to the project. '
-                    'If in a GCS bucket, include full path, e.g. '
-                    '"gs://<bucketname>/path/to/file".')
-
-flags.DEFINE_boolean('dry_run', False,
-                     'If True will simulate the changes and not change any '
-                     'policies.')
-
-flags.DEFINE_integer('concurrent_threads', 10,
-                     'The number concurrent worker threads to use.',
-                     lower_bound=1, upper_bound=50)
-
-flags.DEFINE_integer('maximum_firewall_write_operations', 10,
-                     'The maximum number of in flight write operations on '
-                     'project firewalls. Each running thread is allowed up to '
-                     'this many running operations, so to limit the over all '
-                     'number of operations, limit the number of write threads '
-                     'using the maximum_project_writer_threads flag.',
-                     lower_bound=0, upper_bound=50)
-
-flags.DEFINE_integer('maximum_project_writer_threads', 1,
-                     'The maximum number of projects with active write '
-                     'operations on project firewalls.',
-                     lower_bound=0, upper_bound=50)
-
-# Setup flags
-FLAGS = flags.FLAGS
+ARG_PARSER = argparse.ArgumentParser()
```

Comment:
Should this block of code goes under `if __name__ == '__main__'`?

Reply:
Rather than re-architect `enforcer` I went ahead and did argument parsing at the same level (global) as it was done before.

Reply:
Got it, thanks!